javascript/createElement/js-createelement_toast-messages/js/index.js

```diff
@@ -5,9 +5,14 @@ const addButton = document.querySelector('[data-js="add-button"]');
 const clearButton = document.querySelector('[data-js="clear-button"]');
 
 addButton.addEventListener("click", () => {
+  const newToast = document.createElement("li");
+  newToast.classList.add("toast-container__message");
+  newToast.textContent = "Looking good!";
+  toastContainer.appendChild(newToast);
```

Comment:
Did you talk in class about "appendChild"? As I'm only working with the handout, this is not familiar to me. And it's not necessary from my point of view. 

On the other hand, you are using "toastContainer", but you haven't declared this variable yet.

Reply:
Ah, good to know! Thanks. :) 

Yeah, as I didn't know about appendChild, I did it with append only:

`const toastContainer = document.querySelector('[data-js="toast-container"]');
const addButton = document.querySelector('[data-js="add-button"]');
const clearButton = document.querySelector('[data-js="clear-button"]');

const newToastMessage = document.createElement("toast-container");

addButton.addEventListener("click", () => {
  toastContainer.append(newToastMessage);
  newToastMessage.classList.add("toast-container__message");
  newToastMessage.textContent = "I'm a new toast message";
});`